tests/test_repo_conventions.py

```diff
@@ -0,0 +1,45 @@
+import re
+from pathlib import Path
+from typing import Iterable
+
+import algokit_arc
+import markdown
+import pytest
+from beaker.application import Application
+from bs4 import BeautifulSoup
```

Comment:
Right. Might be worth using the proper package name (look at the pypi page for bs4)

Reply:
Ah thanks, didn't realise the package name was different than the module name